MekHQ/src/mekhq/gui/utilities/WrapLayout.java

```diff
@@ -0,0 +1,192 @@
+package mekhq.gui.utilities;
+
+import java.awt.*;
+import javax.swing.JScrollPane;
+import javax.swing.SwingUtilities;
+
+/**
```

Comment:
If you copy someone else's work, be sure to attribute it to the original author and provide a reference and all pertinent information about terms of use.